packages/node_modules/samples/browser-call-with-screenshare/index.html

```diff
@@ -44,37 +44,65 @@
       <main>
         <h1>Local Screensharing</h1>
         <p>This sample demonstrates sharing your local screen</p>
+
+        <form >
+          <fieldset id="auth-type">
+            <legend>Pick Authentication Method</legend>
+              <div>
+                <input type="radio" id="oauth-input" name="typeCredentials" onchange="handleAuthTypeChange()" value="oauth" checked>
+                <label for="oauth-input">OAuth</label>
+              </div>
+              <div>
+                <input type="radio" id="token-input" name="typeCredentials" onchange="handleAuthTypeChange()" value="token">
+                <label for="token-input">Paste Access Token</label>
+              </div>
+          </fieldset>
+        </form>
+
         <form id="credentials">
-          <fieldset>
-            <legend>Credentials</legend>
-            <input
-              id="access-token"
-              name="accessToken"
-              placeholder="Your access token"
-              type="text">
-            <input id="connect" title="connect" type="submit" value="connect">
-            <p id="connection-status">disconnected</p>
+          <fieldset id="oauth" style="display: block;">
+            <legend>Authentication with OAuth </legend>
+              <input id="login" title="login" type="submit" value="Login">
```

Comment:
Since we are just adding event listeners on these buttons, and not using the form to do the submit, let's change the type on all the inputs to "button"

```suggestion
              <input id="login" title="login" type="button" value="Login">
```